City-of-Pori/pori-kada
# PORI-X Compiled the kada theme styles with production settings, remov…

…ed bugging uglify from visitpori gulp config and compiled visitpori styles with production settings

To test:

1. drush cc all
2. Go to local.pori.fi and local.visitpori.fi and make sure styles work correctly on both.
